### PR TITLE
MTTDOC-293 initial pass for branding

### PR DESCRIPTION
**Pull request by Briancoughlin**
### ****WIP DO NOT FORCE MERGE****

**Select the type of change:**

- [ ] Small Changes - Typos, formatting, slight revisions
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Work on the rebranding of the Documentation to Netcode for GameObjects from MLAPI

**Issue Number:** 
MTTDOC-293

### ****WIP DO NOT FORCE MERGE****

### CODE REVIEW

File: docs/advanced-topics/custom-serialization.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ id: custom-serialization
 title: Custom Serialization
 ---
 
-When using `RPC`'s, `NetworkVariable`'s or any other MLAPI related task that requires serialization. The MLAPI uses a default serialization pipeline that looks like this:
+When using `RPC`'s, `NetworkVariable`'s or any other Netcode for Gameobjects (Netcode) related task that requires serialization. The Netcode uses a default serialization pipeline that looks like this:
```
Comment (MFatihMAR) [inline docs/advanced-topics/custom-serialization.md:6]: Netcode for GameObject is capitalized as **N**etcode for **G**ame**O**bjects with O also being capitalized.
I think you'll need to change that in all markdown docs, unfortunately :)

File: docs/advanced-topics/custom-serialization.md [hunk 2]
```diff
@@ -45,25 +45,25 @@ With this flow, you can override **ALL** serialization for **ALL** types, even b
 To register a custom type, or override an already handeld type you need to register a handler pair.
 
 ```csharp
-// Tells the MLAPI how to serialize and deserialize Url in the future.
+// Tells the  Netcode how to serialize and deserialize Url in the future.
```
Comment (MFatihMAR) [inline docs/advanced-topics/custom-serialization.md:48]: super-nit :)
```suggestion
// Tells the Netcode how to serialize and deserialize Url in the future.
```

File: docs/advanced-topics/custom-transports.md [hunk 3]
```diff
@@ -3,21 +3,21 @@ id: custom-transports
 title: Custom Transports
 ---
 
-The MLAPI uses Unity’s [UNet Transport Layer API ](https://docs.unity3d.com/Manual/UNetUsingTransport.html) by default and has support for custom Transports. 
+Netcode for Gameobjects (Netcode) uses Unity’s [UNet Transport Layer API ](https://docs.unity3d.com/Manual/UNetUsingTransport.html) by default and has support for custom Transports.
```
Comment (MFatihMAR) [inline docs/advanced-topics/custom-transports.md:6]: just a note on that, we might come back to this after UTP becomes the default transport for the SDK.

File: docs/advanced-topics/custom-transports.md [hunk 4]
```diff
@@ -3,21 +3,21 @@ id: custom-transports
...
 
 A Transport collects messages from the application and transmits them over the network. 
 The Transport layer is also responsible for making sure that all packets arrive and that they arrive in order if needed. Transports also make sure that your data won't get corrupted when it's sent over the network.
 
 ## Community Contribution
 
-The MLAPI supports community contributions for Transport implementations. See the [MLAPI Community Contributions](https://github.com/Unity-Technologies/mlapi-community-contributions)  repository to learn more.
+Netcode supports community contributions for Transport implementations. See the [Netcode Community Contributions](https://github.com/Unity-Technologies/mlapi-community-contributions)  repository to learn more.
 
 To use a Transport, follow the [Community Transport Installation Guide](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/Transports).
```
Comment (MFatihMAR) [inline docs/advanced-topics/custom-transports.md:15]: This whole "Community Contributions" section might need a rework since we're going to keep that contribs repo more broad as Multiplayer, not just for the SDK. @LukeStampfli

File: docs/advanced-topics/networktime-ticks.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
+---
+id: networktime-ticks
+title: NetworkTime and Ticks
+sidebar_label: NetworkTime & Ticks
+---
+
+## LocalTime and ServerTime
```
Comment (MFatihMAR) [inline docs/advanced-topics/networktime-ticks.md:7]: this document is a brand-new add, should this go under a separate PR?

File: docs/api-introduction.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
 title: API References
 id: api-introduction
-description: Learn more about the available APIs for Unity Multiplayer Networking, including MLAPI and Transport.
+description: Learn more about the available APIs for Unity Multiplayer Networking, including Netcode for Gameobjects and Transport.
 ---
 
 Multiplayer Networking supports and uses many APIs including the following packages:
 
-* [Unity MLAPI](mlapi-api/introduction.md)
+* [Unity Netcode for Gameobjects](mlapi-api/introduction.md)
```
Comment (MFatihMAR) [inline docs/api-introduction.md:9]: shouldn't we also rename that folder (`mlapi-api/introduction.md`)?

**Comment by Briancoughlin** [comment]
The sandbox has everything in it

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: docs/getting-started/about-mlapi.md [hunk 7]
```diff
@@ -3,23 +3,23 @@ id: about-mlapi
 title: Getting Started with MLAPI
```
Comment (MFatihMAR) [inline docs/getting-started/about-mlapi.md:3]: you might want to rename this document too. I'd suggest `about.md` or `about-netcode.md`

**Comment by Briancoughlin** [comment]
I don’t want to rename folders I would mean lots of redirects and potential for broken links

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: docs/learn/getting-started-boss-room.md [hunk 8]
```diff
@@ -92,11 +92,11 @@ To run multiple instances of the same app, you need to use the command line:
 In contrast to running a local setup, when playing over internet we do not necessarily need a built executable. You can run the game in editor.
 :::
 
-Running the game over internet currently requires either setting up a [Photon Transport for MLAPI](#photon-transport) or using [Port Forwarding](#port-forwarding).
+Running the game over internet currently requires either setting up a [Photon Transport for Netcode](#photon-transport) or using [Port Forwarding](#port-forwarding).
 
 #### Photon Transport
 
-[Photon Transport for MLAPI](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/Transports/com.mlapi.contrib.transport.photon-realtime) uses Photon relay server to facilitate communication between clients and server living on different networks.
+[Photon Transport for Netcode](https://github.com/Unity-Technologies/mlapi-community-contributions/tree/master/Transports/com.mlapi.contrib.transport.photon-realtime) uses Photon relay server to facilitate communication between clients and server living on different networks.
```
Comment (MFatihMAR) [inline docs/learn/getting-started-boss-room.md:99]: IIRC, this thing is now called as "Photon Transport for Netcode for GameObjects" @LukeStampfli

File: docs/mlapi-basics/networkvariable.md [hunk 9]
```diff
@@ -11,7 +11,7 @@ There is a  Video Tutorial covering some of the concepts covered in this page [h
```
Comment (MFatihMAR) [inline docs/mlapi-basics/networkvariable.md:11]: again (picking up on this file arbitrarily), you might consider renaming `docs/mlapi-basics/` folder as `docs/netcode-basics` or `docs/basics`

File: docs/tutorials/goldenpath_series/gp_module_two.md [hunk 10]
```diff
@@ -140,7 +140,7 @@ This section adds a Network Transform component that will move the player.
 
 1. Click **Player** prefab. 
 1. Click **Add Component** in the Inspector Tab.
-1. Select **MLAPI** from the list shown.
+1. Select **Netcode for Gameobjects** from the list shown.
```
Comment (MFatihMAR) [inline docs/tutorials/goldenpath_series/gp_module_two.md:143]: I think this should just be "Netcode" here

File: docs/tutorials/goldenpath_series/mlapi_starting-out.md [hunk 11]
```diff
@@ -19,15 +19,15 @@ In the context of software or information modelling, a Golden Path (sometimes ca
 
 ## Requirements
 
-These tutorials requires an MLAPI-supported version of Unity (2019.4+).
+These tutorials requires an Netcode-supported version of Unity (2019.4+).
```
Comment (MFatihMAR) [inline docs/tutorials/goldenpath_series/mlapi_starting-out.md:22]: actually, 2019.4+ should now be replaced with 2020.3+

**Review by MFatihMAR — CHANGES_REQUESTED** [review]
did my first pass, I think there are improvements and changes needed.

File: docs/advanced-topics/networktime-ticks.md (hunk 5 above)
Comment (Briancoughlin) [reply docs/advanced-topics/networktime-ticks.md:7]: it has a seperate PR. this was me not clearing visual studio when I changed the branch and editing this file by mistake. i can remove if its a problem

**Comment by MFatihMAR** [comment]
@Briancoughlin 
> I don’t want to rename folders I would mean lots of redirects and potential for broken links

well, if there'll ever going to be a good time to rename folders properly, this is the time (before release) — and I think we should.

**Comment by Briancoughlin** [comment]
Closing as Im having an issue with the branch will reopen and will keep these in mind

File: docs/tutorials/goldenpath_series/mlapi_starting-out.md (hunk 11 above)
Comment (Briancoughlin) [reply docs/tutorials/goldenpath_series/mlapi_starting-out.md:22]: Done
